src/models/GraphModel.js

```diff
@@ -0,0 +1,47 @@
+class GraphModel {
+    constructor (metricData, dimensionalData) {
+        this.metricData = metricData;
+        this.dimensionalData = dimensionalData;
+    }
+    getGraphData () {
+        const xAxisValue = 'timestamp';
+        const yAxisValue = 'views';
+        this.dimensionalData.measure('timestamp');
+        const activeBreakdown = this.metricData.breakdowns.filter((breakdown) => {
+            return breakdown.values.some((v) => {
+                return v.on
+            });
+        })[0];
+        if (activeBreakdown) {
```

Comment:
no need for the if, just pass activeBreakdown ? activeBreakdown.breakdownName : undefined as the second column, then you can dry up the code.